src/ecalc/libraries/libecalc/common/libecalc/dto/result/results.py

```diff
@@ -138,6 +139,7 @@ class CompressorModelResult(ConsumerModelResultBase, CompressorTrainResult):
     componentType: Literal[ComponentType.COMPRESSOR]
     requested_inlet_pressure: TimeSeriesFloat
     requested_outlet_pressure: TimeSeriesFloat
+    rate_type_all_rates_incl_stages: RateType
```

Comment:
I think we should remove the CompressorTrainResult and use TimeSeries for those attributes instead. It makes it easier for consumers of the api (webapp) if we have a consistent way of representing the arrays/timeseries/results/values.